clang/lib/StaticAnalyzer/Core/BugReporter.cpp

```diff
@@ -2213,7 +2213,7 @@ void BasicBugReport::Profile(llvm::FoldingSetNodeID& hash) const {
 void PathSensitiveBugReport::Profile(llvm::FoldingSetNodeID &hash) const {
   hash.AddInteger(static_cast<int>(getKind()));
   hash.AddPointer(&BT);
-  hash.AddString(Description);
+  hash.AddString(getShortDescription());
```

Comment:
Is it intentional that you use a member function here instead of the member variable?

Reply:
Yes, it's intentional, because the method call `getShortDescription()` is equivalent to `ShortDescripton.empty() ? Description : ShortDescription`. (The common case when the short and full descriptions are identical is represented internally by an empty `ShortDescription` and the shared value stored in `Description`.)